package.json

```diff
@@ -0,0 +1,34 @@
+{
+  "name": "leaderboard",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "webpack serve",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "build": "webpack --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/Theophileaseh/Leaderboard.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/Theophileaseh/Leaderboard/issues"
+  },
+  "homepage": "https://github.com/Theophileaseh/Leaderboard#readme",
+  "dependencies": {
+    "@babel/core": "^7.17.8",
+    "@babel/preset-env": "^7.16.11",
+    "babel-loader": "^8.2.4",
+    "webpack-cli": "^4.9.2"
+  },
+  "devDependencies": {
+    "css-loader": "^6.7.1",
+    "html-loader": "^3.1.0",
+    "style-loader": "^3.3.1",
+    "webpack-dev-server": "^4.8.0"
+  }
```

Comment:
- [ ] Please install webpack and webpack-cli it should exist in your `devDependencies` to make everything works perfectly with you.